modules/reference/pages/config/server-configuration-overview.adoc

```diff
@@ -194,6 +195,24 @@ For example, if you enter `${my.env.var}` in the `server.xml` file, it looks for
 . my_env_var
 . MY_ENV_VAR
 
+Variables can be loaded from files located in directories that are specified by the `VARIABLE_SOURCE_DIRS` environment variable.
+The default location is `${server.config.dir}/variables`.
```

Comment:
might need a little more context here to make clear what the default is for, something like:

If this environment variable is not specifed, the default location for the variables directory is `${server.config.dir}/variables`.